tests-hivemq-operator/gradle/libs.versions.toml

```diff
@@ -17,12 +16,9 @@ awaitility = { module = "org.awaitility:awaitility", version.ref = "awaitility"
 bouncycastle-pkix = { module = "org.bouncycastle:bcpkix-jdk18on", version.ref = "bouncycastle" }
 bouncycastle-prov = { module = "org.bouncycastle:bcprov-jdk18on", version.ref = "bouncycastle" }
 fabric8-kubernetes-client = { module = "io.fabric8:kubernetes-client", version.ref = "fabric8-kubernetes-client" }
-groovy = { module = "org.apache.groovy:groovy-all", version.ref = "groovy" }
 hivemq-mqttClient = { module = "com.hivemq:hivemq-mqtt-client", version.ref = "hivemq-mqttClient" }
-junit-jupiter = { module = "org.junit.jupiter:junit-jupiter", version.ref = "junit-jupiter" }
 logback-classic = { module = "ch.qos.logback:logback-classic", version.ref = "logback" }
 slf4j-api = { module = "org.slf4j:slf4j-api", version.ref = "slf4j" }
 testcontainers = { module = "org.testcontainers:testcontainers", version.ref = "testcontainers" }
-testcontainers-hivemq = { module = "org.testcontainers:hivemq", version.ref = "testcontainers" }
-testcontainers-junit-jupiter = { module = "org.testcontainers:junit-jupiter", version.ref = "testcontainers" }
+testcontainers-junitJupiter = { module = "org.testcontainers:junit-jupiter", version.ref = "testcontainers" }
```

Comment:
out of curiosity here, why we have some using camelCase and other using kebab-case? For instance we have:

```
selenium-remote-driver
testcontainers-junitJupiter
```

is there any recommendation or what is the best practices to follow here?

Reply:
The hyphen `-` in the versions.toml file can be seen as a separator which gets a dot `.` in the generated code.
For example `testcontainers-hivemq` translates to `testcontainers.hivemq`.
This separation should only be done where it makes sense. The default should be camelCase, and the hypen should only be used at places where it groups certain dependencies.

In the specific case of `testcontainers.junit.<xxx>` there is no other dependency than jupiter for the `<xxx>` part.

Related blog post: https://melix.github.io/blog/2021/03/version-catalogs-faq.html#_should_i_use_commons_lang3_as_an_alias_or_commonslang3